cmd/meroxa/root/connectors/connect_test.go

```diff
@@ -174,3 +174,16 @@ Source connector %q and destination connector %q successfully created!
 		t.Fatalf("expected \"%v\", got \"%v\"", connectors, gotConnectors)
 	}
 }
+
+func TestConnectExecutionNoFlags(t *testing.T) {
```

Comment:
This wasn't changed on this PR, but I realized I should have added this test before.